tests/flow/test_distinct.py

```diff
@@ -1,8 +1,75 @@
-from redis_base import RedisGraphTestBase
+from redisgraph import Graph, Node, Edge
 
-class ReturnDistinctFlowTest_1(RedisGraphTestBase):
+from base import FlowTestsBase
+
+redis_graph = None
+redis_con = None
+
+class RedisGraphTestBase(FlowTestsBase):
+    def __init__(self):
+        super(RedisGraphTestBase, self).__init__()
+        global redis_con
+        global redis_graph
+
+        redis_con = self.env.getConnection()
+        self.create_graph()
     @classmethod
-    def createCommands(cls):
+    def graphId(cls):
+        return cls.__name__
+
+    def createCommand(self):
```

Comment:
Maybe rise exception here as this function is meant to be overwritten by subclass.

Reply:
done